client/x/evmengine/keeper/abci_internal_test.go

```diff
@@ -407,31 +407,33 @@ func TestKeeper_PostFinalize(t *testing.T) {
 			},
 			postStateCheck: payloadFailedToSet,
 		},
-		{
-			name: "fail: unknown status from EL",
-			mockEngine: mockEngineAPI{
-				forkchoiceUpdatedV3Func: func(ctx context.Context, update eengine.ForkchoiceStateV1,
-					payloadAttributes *eengine.PayloadAttributes) (eengine.ForkChoiceResponse, error) {
-					return eengine.ForkChoiceResponse{
-						PayloadStatus: eengine.PayloadStatusV1{
-							Status:          "unknown status",
-							LatestValidHash: nil,
-							ValidationError: nil,
-						},
-						PayloadID: &payloadID,
-					}, nil
+		/*
+			{
+				name: "fail: unknown status from EL",
```

Comment:
Commented this test out in this [commit](https://github.com/piplabs/story/pull/372/commits/b9e0b91f8345dcce7a26ad34611e7a11804c2dd6) as unknown status of payload is not handled anymore.

Reply:
Should we just delete it?

Reply:
I left it just in case, but I think we can delete it.

Reply:
deleted in [d37a1e0](https://github.com/piplabs/story/pull/372/commits/d37a1e02eb93b66ca251ba38a5c6c99df82d544a).